src/components/scenes/Fio/FioDomainRegisterSelectWalletScene.tsx

```diff
@@ -99,12 +99,7 @@ class FioDomainRegisterSelectWallet extends React.PureComponent<Props, LocalStat
     const { supportedAssets } = this.state
 
     const result = await Airship.show<WalletListResult>(bridge => (
-      <WalletListModal
-        bridge={bridge}
-        navigation={this.props.navigation}
-        headerTitle={lstrings.select_wallet}
-        allowedAssets={[...supportedAssets, { pluginId: 'fio', tokenId: null }]}
-      />
+      <WalletListModal bridge={bridge} navigation={this.props.navigation} headerTitle={lstrings.select_wallet} allowedAssets={supportedAssets} />
```

Comment:
why was FIO removed?

Reply:
https://edgesecure.slack.com/archives/C01GGKCV6NM/p1714680544592859?thread_ts=1714593367.028079&cid=C01GGKCV6NM

Reply:
Still need to support FIo since we activate w/FIO directly on chain w/o the API